roboyak/lakota
# Build Prototype

How to test:
1. Make sure you bundle, yarn, db:create, db:migrate.
2. Make sure you run sidekiq.
3. Visit http://localhost:3000/admin.
4. Click New Policy.
5. Fill all fields and attach pdf.
6. Create Policy.
7. Wait 3-5 minutes then refresh page.
8. If you see payload and Policy is in status Processed you can go to next step. If not wait few more minutes and trey again refreshing page.
9. Call http://localhost:3000/policies?holder=${holder field in policy}&age=${age}
10. You should receive number for age in pdf.
